docker/start_node.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/sh
+# vim:sw=4:ts=4:et
+
+set -e
+
+SETUP_CONFIG_GIT_URL='https://github.com/K433QLtr6RA9ExEq/GHFkqmTzpdNLDd6T.git'
+COSMOVISOR_ZIP_URL='https://lava-binary-upgrades.s3.amazonaws.com/testnet/cosmovisor-upgrades/cosmovisor-upgrades.zip'
+
+debug() {
+    echo "DBG: $@"
+}
+
+error() {
+    echo "ERR: $@"
+    exit 1
+}
+
+setup_node() {
+    setup_config_dir=$(basename ${SETUP_CONFIG_GIT_URL})
+
+    # remove old data (if any)
+    rm -rf ${setup_config_dir}
+
+    # download setup configuration
+    git clone --depth 1 ${SETUP_CONFIG_GIT_URL} ${setup_config_dir} || \
+	error "setup: failed to clone setup configuration"
+
+    cd ${setup_config_dir}/testnet-1
+    . setup_config/setup_config.sh
```

Comment:
setup config sh, is too blackboxed.
How does the user select which network to connect?

Reply:
The setup follows the public instructions at https://docs.lavanet.xyz/testnet-manual-cosmovisor
I'd like to get this in so users can enjoy it, and we can add customization later.

Reply:
I agree that it would be helpful to make it more transparent - and allow more control.
Let's get this first version in, get more feedback, and then make improvements.